src/main/java/tech/autodirect/api/database/TableOffers.java

```diff
@@ -189,20 +188,18 @@ public void markOfferUnclaimed(int offerId) throws SQLException, ResponseStatusE
         stmt.close();
     }
 
-    public boolean dropTable() throws SQLException {
-        return dropTable(this.tableName);
+    public void dropTable() throws SQLException {
```

Comment:
@Shalev-Lifshitz we can merge this PR if you can confirm that you don't have code that depends on this returning a boolean.

Reply:
@sammdu All tests pass so we should be fine.